Examples/Tests/Langmuir/analysis_langmuir_multi_2d.py

```diff
@@ -35,88 +42,116 @@
 # Parse test name and check if Vay current deposition (algo.current_deposition=vay) is used
 vay_deposition = True if re.search( 'Vay_deposition', fn ) else False
 
-# Parameters (these parameters must match the parameters in `inputs.multi.rt`)
+# Parameters (these parameters must match the parameters in the relevant input files as well as the
+# additional parameters specified as runtime_params for the relevant tests in Regression/WarpX-tests.ini)
 epsilon = 0.01
 n = 4.e24
 n_osc_x = 2
 n_osc_z = 2
 xmin = -20e-6; xmax = 20.e-6; Nx = 128
 zmin = -20e-6; zmax = 20.e-6; Nz = 128
 
-# Wave vector of the wave
-kx = 2.*np.pi*n_osc_x/(xmax-xmin)
-kz = 2.*np.pi*n_osc_z/(zmax-zmin)
-# Plasma frequency
-wp = np.sqrt((n*e**2)/(m_e*epsilon_0))
+Lx = xmax - xmin
+Lz = zmax - zmin
 
-k = {'Ex':kx, 'Ez':kz}
-cos = {'Ex': (0,1,1), 'Ez':(1,1,0)}
+dx = Lx / Nx
+dz = Lz / Nz
 
-def get_contribution( is_cos, k ):
-    du = (xmax-xmin)/Nx
-    u = xmin + du*( 0.5 + np.arange(Nx) )
-    if is_cos == 1:
-        return( np.cos(k*u) )
-    else:
-        return( np.sin(k*u) )
```

Comment:
This function, now removed, was using always the parameters along x (`xmin`, `xmax`, `Nx`), for both the x and z contributions, which would be potentially wrong if we were to set up some of the parameters in a way that is not symmetric.